examples/00-fluent/mixing_elbow_settings_api.py

```diff
@@ -180,7 +180,8 @@
 # Solve for 150 iterations.
 
 solver.solution.run_calculation.iterate.get_attr("arguments")
-solver.solution.run_calculation.iterate(number_of_iterations=150)
```

Comment:
Hi @prmukherj. How does the fix work? If this line doesn't work, is that a bug and is an issue raised?  Thanks.

Reply:
@seanpearsonuk , I am checking why it is failing. Locally I could reproduce the issue and the solution that I have posted works locally. If it fixes the nightly docs build, we can investigate further I think.

Reply:
@seanpearsonuk and @mkundu1 , Thank you for clarifying this. It has been fixed. This looks good?